SetupDataPkg/ConfApp/DfciUsb/DfciUsb.c

```diff
@@ -23,104 +21,8 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Library/UefiBootServicesTableLib.h>
 #include <Library/PrintLib.h>
 
-#include "DfciUtility.h"
 #include "DfciUsb.h"

```

Comment:
Change the name from Dfci for these files and dir?

Reply:
Replaced with SvdUsb.